src/components/modals/WalletListMenuModal.tsx

```diff
@@ -175,6 +174,11 @@ export function WalletListMenuModal(props: Props) {
       const { pluginIds, label, value } = option
 
       if (Array.isArray(pluginIds) && !pluginIds.includes(pluginId)) continue
+
+      // Special case for `manageTokens`. Only allow pluginsIds that have metatokens
+      if (value === 'manageTokens') {
+        if (account.currencyConfig[pluginId].currencyInfo.metaTokens.length === 0) continue
```

Comment:
Instead of going to the deprecated `currencyConfig[pluginId].currencyInfo.metaTokens.length`, why not go to `Object.keys(currencyConfig[pluginId].builtinTokens).length`? That is the modern thing, and the core keeps it exactly in sync with `metaTokens`.

Reply:
That's fair. This was coded up before your comment for the core. will do